### PR TITLE
OCPBUGS-36858: Update openshift-e2e-openstack-dualstack worker replicas 2

### PR DESCRIPTION
**Pull request by MiguelCarpio**
DualStack Periodic jobs failed due to `[sig-architecture] Conformance Tests should have at least two untainted nodes`. The test needs two compute nodes, but the job is run with only one.

### CODE REVIEW

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.15-e2e-openstack-additional-ipv6-network

File: ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh [hunk 1]
```diff
@@ -31,6 +31,10 @@ if [[ -n "$ADDITIONAL_SECURITY_GROUP_RULES" ]]; then
 					echo "Adding ${service} rule to security group ${sg_id}" 
 					openstack security group rule create "$sg_id" --protocol tcp --dst-port 12865:12865 --remote-ip 0.0.0.0/0
 					;;
+				IPv6DualStack)
+					echo "Adding ${service} rule to security group ${sg_id}" 
+					openstack security group rule create "$sg_id" --protocol tcp --dst-port 9080:9080 --remote-ip 0.0.0.0/0
```
Comment (MaysaMacedo) [inline ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh]: why is this needed? Depending on the reason it might be worth to add a comment

**Comment by MaysaMacedo** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Review by mandre — COMMENTED** [review]
Alternatively, we can chose to ignore the `[sig-architecture] Conformance Tests should have at least two untainted nodes` test for the dual stack job that we configure with only one worker.

Ideally, the test should know there's only one worker... We should look at the test and fix it.

File: ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh (hunk 1 above)
Comment (MiguelCarpio) [reply ci-operator/step-registry/openstack/provision/securitygroups/openstack-provision-securitygroups-commands.sh]: Hello, Don't worry. It was a test, I've removed it)

**Comment by MiguelCarpio** [comment]
@mandre @MaysaMacedo After adding a second worker, we failed at `[sig-network] [Feature:IPv6DualStack] Granular Checks: Services Secondary IP Family [LinuxOnly] should function for service endpoints using hostNetwork [Suite:openshift/conformance/parallel] [Suite:k8s]` I'll take a look on it; however, do you have any idea about how to solve it or, indeed, is this a potential risk in dualstack env?

**Comment by mandre** [comment]
> @mandre @MaysaMacedo After adding a second worker, we failed at `[sig-network] [Feature:IPv6DualStack] Granular Checks: Services Secondary IP Family [LinuxOnly] should function for service endpoints using hostNetwork [Suite:openshift/conformance/parallel] [Suite:k8s]` I'll take a look on it; however, do you have any idea about how to solve it or, indeed, is this a potential risk in dualstack env?

Yep, that test should be investigated.
From [the logs](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55706/rehearse-55706-periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack/1825608285642821632) we see:
```
fail [k8s.io/kubernetes/test/e2e/network/dual_stack.go:669]: failed dialing endpoint, did not find expected responses... 
Tries 34
Command curl -g -q -s 'http://10.128.2.72:9080/dial?request=hostname&protocol=http&host=fd02::5398&port=80&tries=1'
retrieved map[ip3m8ysi-2d6fc-d76w6-worker-0-hpt2d:{}]
expected map[ip3m8ysi-2d6fc-d76w6-worker-0-5mfsn:{} ip3m8ysi-2d6fc-d76w6-worker-0-hpt2d:{}]
Error: exit with code 1
```

We should look at the test see what it does.

**Comment by MaysaMacedo** [comment]
@MiguelCarpio can you squash all commits into one?
After that, this PR LGTM.

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.17-e2e-openstack-ovn-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-e2e-openstack-dualstack periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-dualstack periodic-ci-shiftstack-ci-release-4.15-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-e2e-openstack-dualstack periodic-ci-shiftstack-ci-release-4.15-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.18-e2e-openstack-dualstack

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.15-e2e-openstack-dualstack

**Comment by MaysaMacedo** [comment]
/lgtm
For the future, in the commit message it's worth mentioning the changes are for openstack
